inseq/attr/feat/perturbation_attribution.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Dict
+
+import logging
+
+from captum.attr import Occlusion
+
+from ...data import PerturbationFeatureAttributionStepOutput
+from ...utils import Registry
+from ..attribution_decorators import set_hook, unset_hook
+from .attribution_utils import get_source_target_attributions
+from .gradient_attribution import FeatureAttribution
+
+
+logger = logging.getLogger(__name__)
+
+
+class PerturbationMethodRegistry(FeatureAttribution, Registry):
```

Comment:
Let's call it `PerturbationAttribution` to keep it consistent with `GradientAttribution`. We might want to bulk change them to add the `Registry` specification at a later time though!

Reply:
Let's call it PerturbationAttribution to keep it consistent with GradientAttribution. We might want to bulk change them to add the Registry specification at a later time though!